app/assets/images/blacklight/compact.svg

```diff
@@ -1 +1,5 @@
+<!--
```

Comment:
My understanding was that this came from Material Icons. I'm not sure we need additional coments

Reply:
Fixed.